source/jobs/main.rst

```diff
@@ -78,11 +60,32 @@ Master's Topics
   the region of interest.  The impacts of homogenization on R2S
   activation workflows is a natural extension of this work.
 
-* **Variance Reduction in Complex 3-D Geometries** This will extend
-  our collaboration with ORNL on their CADIS/FW-CADIS methods, but
-  applying them to CAD-based geometries and consideration of complex
-  source distributions, using knowledge of nuclear engineering with
-  some applied math.
+Undergraduate Opportunities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CNERG is happy to provide opportunities for interested undergraduates to
+become involved in our computational research.  Like most research groups,
+there are frequently tasks and assignments that are appropriate for an
+undergraduate who is just developing their research skills.  The most
+successful undergraduate researchers have clear expectations for their work as
```

Comment:
I am not sure what you mean by "clear expectations". In terms of what? I imagine this statement could be intimidating for some.